packages/faceted-search/src/components/FacetedToolbar/FacetedToolbar.component.js

```diff
@@ -1,33 +1,22 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import Toggle from '@talend/react-components/lib/Toggle';
+import { Form } from '@talend/design-system';
 import { getTheme } from '@talend/react-components/lib/theme';
 
 import { FACETED_MODE, USAGE_TRACKING_TAGS } from '../../constants';
 import cssModule from './FacetedToolbar.scss';
 
 const theme = getTheme(cssModule);
 
-const SwitchFacetedMode = ({ facetedMode, onChange, t }) => {
-	const values = [
-		{
-			value: FACETED_MODE.BASIC,
-			label: t('FACETED_SEARCH_BASIC', { defaultValue: 'Basic' }),
-			dataFeature: USAGE_TRACKING_TAGS.BASIC,
-		},
-		{
-			value: FACETED_MODE.ADVANCED,
-			label: t('FACETED_SEARCH_ADVANCED', { defaultValue: 'Advanced' }),
-			dataFeature: USAGE_TRACKING_TAGS.ADVANCED,
-		},
-	];
-
-	return (
-		<form>
-			<Toggle.Label values={values} value={facetedMode} onChange={onChange} />
-		</form>
-	);
-};
+const SwitchFacetedMode = ({ facetedMode, onChange, t }) => (
+	<form>
+		<Form.Switch
+			label={t('FACETED_SEARCH_QUERY', 'Query')}
+			onChange={() => onChange(facetedMode === FACETED_MODE.BASIC ? FACETED_MODE.ADVANCED : FACETED_MODE.BASIC)}
+			dataFeature={facetedMode === FACETED_MODE.BASIC ? USAGE_TRACKING_TAGS.ADVANCED : USAGE_TRACKING_TAGS.BASIC}
+		/>
+	</form>
```

Comment:
You can use the Form component from the DS too
```suggestion
	<Form>
		<Form.Switch
			label={t('FACETED_SEARCH_QUERY', 'Query')}
			onChange={() => onChange(facetedMode === FACETED_MODE.BASIC ? FACETED_MODE.ADVANCED : FACETED_MODE.BASIC)}
			dataFeature={facetedMode === FACETED_MODE.BASIC ? USAGE_TRACKING_TAGS.ADVANCED : USAGE_TRACKING_TAGS.BASIC}
		/>
	</Form>
```